java/ql/src/semmle/code/java/frameworks/apache/Collections.qll

```diff
@@ -138,127 +188,420 @@ private class ApacheKeyValueModel extends SummaryModelCsv {
+          ";CollectionUtils;true;collate;;;Element of Argument[1];Element of ReturnValue;value",
+          ";CollectionUtils;true;disjunction;;;Element of Argument[0];Element of ReturnValue;value",
+          ";CollectionUtils;true;disjunction;;;Element of Argument[1];Element of ReturnValue;value",
+          ";CollectionUtils;true;emptyIfNull;;;Argument[0];ReturnValue;value",
+          ";CollectionUtils;true;extractSingleton;;;Element of Argument[0];ReturnValue;value",
+          ";CollectionUtils;true;find;;;Element of Argument[0];ReturnValue;value",
+          ";CollectionUtils;true;get;(Iterator,int);;Element of Argument[0];ReturnValue;value",
+          ";CollectionUtils;true;get;(Iterable,int);;Element of Argument[0];ReturnValue;value",
+          ";CollectionUtils;true;get;(Map,int);;MapKey of Argument[0];MapKey of ReturnValue;value",
+          ";CollectionUtils;true;get;(Map,int);;MapValue of Argument[0];MapValue of ReturnValue;value",
+          ";CollectionUtils;true;get;(Object,int);;ArrayElement of Argument[0];ReturnValue;value",
+          ";CollectionUtils;true;get;(Object,int);;Element of Argument[0];ReturnValue;value",
+          ";CollectionUtils;true;get;(Object,int);;MapKey of Argument[0];MapKey of ReturnValue;value",
+          ";CollectionUtils;true;get;(Object,int);;MapValue of Argument[0];MapValue of ReturnValue;value",
+          ";CollectionUtils;true;getCardinalityMap;;;Element of Argument[0];MapKey of ReturnValue;value",
+          ";CollectionUtils;true;intersection;;;Element of Argument[0];Element of ReturnValue;value",
+          ";CollectionUtils;true;intersection;;;Element of Argument[1];Element of ReturnValue;value",
+          ";CollectionUtils;true;permutations;;;Element of Argument[0];Element of Element of ReturnValue;value",
+          ";CollectionUtils;true;predicatedCollection;;;Element of Argument[0];Element of ReturnValue;value",
+          ";CollectionUtils;true;removeAll;;;Element of Argument[0];Element of ReturnValue;value",
+          ";CollectionUtils;true;retainAll;;;Element of Argument[0];Element of ReturnValue;value",
+          ";CollectionUtils;true;select;;;Element of Argument[0];Element of ReturnValue;value",
+          ";CollectionUtils;true;select;;;Element of Argument[0];Element of Argument[2];value",
+          ";CollectionUtils;true;select;;;Element of Argument[0];Element of Argument[3];value",
+          ";CollectionUtils;true;selectRejected;;;Element of Argument[0];Element of ReturnValue;value",
+          ";CollectionUtils;true;selectRejected;;;Element of Argument[0];Element of Argument[2];value",
+          ";CollectionUtils;true;subtract;;;Element of Argument[0];Element of ReturnValue;value",
+          ";CollectionUtils;true;synchronizedCollection;;;Element of Argument[0];Element of ReturnValue;value",
+          ";CollectionUtils;true;transformingCollection;;;Element of Argument[0];Element of ReturnValue;value",
```

Comment:
We haven't modelled other collection operations that apply some lambda; for consistency we probably shouldn't model this one either

Reply:
But the lambda isn't applied by this function: `Existing entries in the specified collection will not be transformed`.

Reply:
I'll add a comment like you ask for below.

Reply:
Ah yes, comment will do the trick